python/lsst/ts/electrometer/controller.py

```diff
@@ -55,7 +55,7 @@ class ElectrometerController:
 
     """
 
-    def __init__(self, log=None):
+    def __init__(self, csc, log=None):
```

Comment:
At line 49 and 51: manual_start_time and manual_end_time are float, not int. And if you change them to TAI (requested below) then please document the units as TAI seconds.

Reply:
Done by me since Eric is on vacation.